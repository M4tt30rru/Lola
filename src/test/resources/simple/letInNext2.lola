##Import "std.lola"
##Find let ##Identifier(_x) <- ##Literal(_i) in ##Identifier(_col);
 ##replace ##(_x) = ##(_col).get(##(_i));

let x <- 4 in list;
