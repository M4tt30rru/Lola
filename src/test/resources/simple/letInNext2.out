x = list.get(4);
